ERCS/erc-5564.md

```diff
@@ -4,7 +4,8 @@ title: Stealth Addresses
 description: Private, non-interactive transfers and interactions
 author: Toni Wahrstätter (@nerolation), Matt Solomon (@mds1), Ben DiFrancesco (@apbendi), Vitalik Buterin (@vbuterin)
 discussions-to: https://ethereum-magicians.org/t/eip-5566-stealth-addresses-for-smart-contract-wallets/10614
-status: Review
+status: Last Call
+last-call-deadline: 2024-03-31
```

Comment:
```suggestion
last-call-deadline: 2024-04-23
```

Giving some extra room for back-and-forth on this PR.

Reply:
Updated in [7e3e4e5](https://github.com/ethereum/ERCs/pull/322/commits/7e3e4e585861a32d59978d3a159f6773afc28992)